From: Juergen Beisert <j.beisert@pengutronixde>
Subject: [libmad] remove special handling for architectures

The configure script adds certain -march flags for different
architectures, which contain a-priori implications which might not be
true for several systems.

Signed-off-by: Juergen Beisert <j.beisert@pengutronixde>
---
--- configure.orig	2006-12-21 18:53:25.000000000 +0100
+++ configure	2006-12-21 18:53:50.000000000 +0100
@@ -1857,10 +1857,9 @@
     if test -z "$arch"
     then
 	case "$host" in
-	    i386-*)           ;;
-	    i?86-*)           arch="-m486" ;;
-	    arm*-empeg-*)     arch="-march=armv4 -mtune=strongarm1100" ;;
-	    armv4*-*)         arch="-march=armv4 -mtune=strongarm" ;;
+	    i?86-*)           ;;
+	    arm*-empeg-*)     ;;
+	    armv4*-*)         ;;
 	    powerpc-*)        ;;
 	    mips*-agenda-*)   arch="-mcpu=vr4100" ;;
 	    mips*-luxsonor-*) arch="-mips1 -mcpu=r3000 -Wa,-m4010" ;;
